core/plugins/org.polarsys.capella.core.platform.sirius.sirius.validation/plugin.properties

```diff
@@ -13,20 +13,20 @@
 #
 # Rules CO21 first resolver.
 #
-quickFix.I_22_Resolver.label = Update all hyperLink to capella element or diagram
-quickFix.I_22_Resolver.desc = Update all hyperLink to capella element or diagram
+quickFix.I_22_Resolver.label = Update displayed text for related hyperlink(s)
+quickFix.I_22_Resolver.desc = Update text of all related hyperlink(s)

```

Comment:
Maybe "hyperlinks" without the (s) because when you are reading without the s, there is no sense to put "all"